File: cpp/Code/Assignment.cpp
#pragma once //untuk mencegah multiple inclusion
#include <string>

using namespace std;

class Assignment {
private:
    // Atribut
    string title;
    string dueDate;

public:
    // Konstruktor
    Assignment() {}
    Assignment(string t, string d) : title(t), dueDate(d) {}

    // Destruktor
    ~Assignment() {}

    // getter untuk title dan dueDate
    string getTitle() const { return title; }
    void setTitle(string t) { title = t; }

    string getDueDate() const { return dueDate; }
    void setDueDate(string d) { dueDate = d; }
};

File: java/Code/Assignment.java
public class Assignment {
    //atribut
    private String title;
    private String dueDate;

    //constructor
    public Assignment(String title, String dueDate) {
        this.title = title;
        this.dueDate = dueDate;
    }

    //getter title
    public String getTitle() {
        return title;
    }

    //getter due date
    public String getDueDate() {
        return dueDate;
    }

    //setter title
    public void setTitle(String title) {
        this.title = title;
    }
    
    //setter due date
    public void setDueDate(String dueDate) {
        this.dueDate = dueDate;
    }
}
